src/perform_step/low_order.jl

```diff
@@ -188,18 +188,17 @@ end
 @muladd function perform_step!(integrator,cache::RKMilCommuteCache,f=integrator.f)
   @unpack du1,du2,K,gtmp,L = cache
   @unpack t,dt,uprev,u,W,p = integrator
-  @unpack I,mil_correction,Kj,Dgj,tmp = cache
+  @unpack J,mil_correction,Kj,Dgj,tmp = cache
   dW = W.dW; sqdt = integrator.sqdt
   f = integrator.f; g = integrator.g
 
   ggprime_norm = 0.0
 
   @.. mil_correction = zero(u)
-  for i=1:length(dW),j=1:length(dW)
-      I[j,i] = 0.5*dW[i]*dW[j]
-      if alg_interpretation(integrator.alg) == :Ito
-        j == i && (I[i,i] -= 0.5*dt) # Ito correction
-      end
+  if alg_interpretation(integrator.alg) == :Ito
+    J = 0.5 .* (vec(dW) .* vec(dW)' - dt .* I)
```

Comment:
This is allocating, and doesn't properly fuse because of the missing `.` on the `-`.